lambdas/accesstoken/src/test/java/uk/gov/di/ipv/cri/address/api/handler/AccessTokenHandlerTest.java

```diff
@@ -55,18 +55,25 @@ void setUp() {
     @Test
     void shouldReturnAccessTokenOnSuccessfulExchange() throws Exception {
         APIGatewayProxyRequestEvent event = new APIGatewayProxyRequestEvent();
+        String authCodeValue = "12345";
+        String grantType = "authorization_code";
         String tokenRequestBody =
-                "code=12345&redirect_uri=http://test.com&grant_type=authorization_code&client_id=test_client_id";
+                String.format(
+                        "code=%s"
+                                + "&client_assertion=%s"
+                                + "&client_assertion_type=urn:ietf:params:oauth:client-assertion-type:jwt-bearer"
+                                + "&client_id=urn:uuid:ipv-core"
+                                + "&grant_type=%s",
```

Comment:
This is really weird as you are using the `String.format` you shouldn't have to use `+`, I think you can replace the `+` with a commas for the same effect

Reply:
I think you can use either `+` or `,` depending on the need. The `+` is just a string concatenation operator in this case and it is there to split the string. We can put all the strings one line. Using `,` the same thing can be achieved but the way the params need to be passed will have to change.